jvm/src/test/kotlin/org/bitcoindevkit/JvmLibTest.kt

```diff
@@ -27,7 +27,7 @@ class JvmLibTest {
     val descriptor =
         "wpkh([c258d2e4/84h/1h/0h]tpubDDYkZojQFQjht8Tm4jsS3iuEmKjTiEGjG6KnuFNKKJb5A6ZUCUZKdvLdSDWofKi4ToRCwb9poe1XdqfUnP4jaJjCB2Zwv11ZLgSbnZSNecE/0/*)"
 
-    val databaseConfig = DatabaseConfig.Memory("")
+    val databaseConfig = DatabaseConfig.Memory
```

Comment:
Does it make sense to also update the README with similar changes as part of this PR?

Reply:
yes! good catch, will fix.